config/create-gke-manifest.js

```diff
@@ -659,68 +656,6 @@ async function applyApiServer() {
                     }
                   }
                 },
-                {
-                  name: 'OAUTH_GOOGLE_DISCOVERY_URL',
```

Comment:
Should we really disable oauth with google? im my opinion we should just delete OAUTH_WEPUBLISH... Variables

Reply:
I left it removed as I will remove this deployment soon anyway. Noone uses this feature and for now local dev is enough.